Makefile

```diff
@@ -617,6 +621,11 @@ metal: examples/metal/metal.cpp ggml.o $(OBJS)
 	$(CXX) $(CXXFLAGS) $^ -o $@ $(LDFLAGS)
 endif
 
+ifeq ($(UNAME_S),Darwin)
+swift: examples/swift
+	cd examples/swift && make build && cd -
```

Comment:
I think rename the example folder to `batched.swift` will be better.

Reply:
✅